src/sql/sqlops.d.ts

```diff
@@ -5,6 +5,7 @@
 
 declare module 'sqlops' {
 	import * as vscode from 'vscode';
+	import { OperatingSystem } from 'vs/base/common/platform';
```

Comment:
we don't add new apis to sqlops.d.ts, its in maintenance mode, any new apis should only be added to the azdata.d.ts

Reply:
Should I revert the changes to sqlops.d.ts then? I can't add DefaultValueOsOverride type without this.

Reply:
No changes should be made to this file.

Reply:
Unfortunately, sqlExtHost.api.impl.ts mixes both sqlops and azdata types together with its own types. There are declarations of the AzureResource enum in _all three places_ ! The result is that if you remove the enum for OssRdbms from sqlops.d.ts, it breaks sqlExtHost.api.impl.ts. If you revert the change in that file, it complains because it is incompatible with azdata.d.ts. In short, the enums have to be in sync across all three files. What I did was to revert all the other changes, but I did still have to add the enum value AzureResource.OssRdbms to sqlops.d.ts. There is currently no way around that without refactoring more code. If that is required, I will need your help to figure out how to do it.

Reply:
Changed reverted from sqlops.d.ts. Added AzureResource.OssRdbms enum to sqlops.proposed.d.ts.